migrations/13_SCHEMES_GenericScheme.js

```diff
@@ -9,10 +9,12 @@ const DxController = artifacts.require('DxController')
 
 const registerScheme = require('./helpers/registerScheme')
 
-module.exports = async function (deployer) {
+module.exports = async function (deployer, network) {
   const dxAvatar = await DxAvatar.deployed()
   const dxController = await DxController.deployed()
 
+  const getDXContractAddress = require('../src/helpers/getDXContractAddresses.js')(network, artifacts)
```

Comment:
You can give the util a web3 method so it's easier to get the network id

note that web3 is globally accessible in `13_SCHEMES_GenericScheme.js` and can be hand it to the util.